src/wagtail_tinytableblock/templates/wagtail_tinytableblock/table_block.html

```diff
@@ -16,7 +16,7 @@
         {% for row in value.data.rows %}
             <tr>
                 {% for cell in row %}
-                    <{{ cell.type }}{% if cell.scope %} scope="{{ cell.scope }}"{% endif %}{% if cell.colspan %} colspan="{{ cell.colspan }}"{% endif %}{% if cell.rowspan %} rowspan="{{ cell.rowspan }}" {% endif %}>{{ cell.value }}</{{ cell.type }}>
+                    <{{ cell.type }}{% if cell.scope %} scope="{{ cell.scope }}"{% endif %}{% if cell.colspan %} colspan="{{ cell.colspan }}"{% endif %}{% if cell.rowspan %} rowspan="{{ cell.rowspan }}" {% endif %}>{{ cell.value|safe }}</{{ cell.type }}>
```

Comment:
somewhat wary of doing this even with the expanded nh3 sanitisation 🤔

Reply:
I think it's ok - `nh3` ought to catch most potential issues. Given we plan intend to render it as HTML, it ought to be ok.

You've got a test to check sanitization works, so it should be ok.